libkbfs/block_types.go

```diff
@@ -190,6 +225,27 @@ func (fb FileBlock) DeepCopy(codec kbfscodec.Codec) (*FileBlock, error) {
 	return &fileBlockCopy, nil
 }
 
+// UpdateHash updates the hash of this FileBlock
+func (fb *FileBlock) UpdateHash() kbfshash.RawDefaultHash {
```

Comment:
Yeah I figured, but I didn't want to change the functionality in case I broke something. I'll rework it to do what you suggested.